docs/getting-started/provision-infrastructure.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Provision Infrastructure
+toc: true
+weight: 3
+indent: true
+---
+
+# Provision Infrastructure
+
+Crossplane's core feature is provisioning infrastructure anywhere using the
+Kubernetes API. Once you have [installed a provider] and [configured your
+credentials], you can create any infrastructure resource currently supported by
+the provider. Let's start by provisioning a database on your provider of choice.
+
+Each provider below offers their own flavor of a managed database. When the
+provider is installed into your Crossplane cluster, it installs a cluster-scoped
+CRD that represents the managed service offering, as well as controllers that
+know how to create, update, and delete instances of the service on the cloud
+provider.
+
+Feel free to skip the section for any provider that you are not interested in
+utilizing.
+
+- [AWS]
+- [GCP]
+- [Azure]
+- [Alibaba]
+
+## AWS
+
+The AWS provider supports provisioning an [RDS] instance with the `RDSInstance`
+CRD it installs into your cluster. Create a new file named `aws-rds.yaml` with
+the following contents:
+```yaml
+apiVersion: database.aws.crossplane.io/v1beta1
+kind: RDSInstance
+metadata:
+  name: rdsmysql
+spec:
+  forProvider:
+    dbInstanceClass: db.t2.small
+    masterUsername: masteruser
+    allocatedStorage: 20
+    engine: mysql
+    skipFinalSnapshotBeforeDeletion: true
+  writeConnectionSecretsToRef:
+    namespace: crossplane-system
+    name: aws-rdsmysql-conn
+  providerRef:
+    name: aws-provider
+  reclaimPolicy: Delete
+```
+
+Create the `RDSInstance`:
+```
+kubectl apply -f aws-rds.yaml
+```
+
+You can view the progress of provisioning the database with the following
+command:
+```
+kubectl get -f aws-rds.yaml
```

Comment:
@muvaf `get` will only show the printer columns right?

Reply:
Oh sorry I thought it was YAML output, which is what I do usually.

Reply:
Is it worth including the `kubectl describe` output anywhere in this guide? I like the succinctness of `kubectl get`, especially when you're just waiting for a thing, but `kubectl describe` (with status, events, etc) communicates much more of a "hey this actually got provisioned" experience in my opinion.

That all said, I could be convinced that `kubectl describe` output would be too overwhelming for this guide.

Reply:
Would you want to `kubectl describe` and show the output here?

Reply:
We could have some output that is collapsible so we could say something like "For more information run `kubectl describe` and you will see output like the following:" <collapsed-content>. That being said, I feel like we keep going back and forth between "all users will know how to use the Kubernetes API" and "some users may not know how to use the Kubernetes API". If we are saying the former then I think users will naturally `kubectl describe` if they want more information.

Reply:
Yeah, I agree it's safe to assume folks will know about `kubectl describe`. My suggestion is not so much because I'm worried the reader won't know they _can_ use `kubectl describe`, but because I think `kubectl describe` makes a stronger impression because (at least to me) there's more "wow factor" in seeing the spec and status of the resource updated to reflect the state in the cloud. That all said, it is quite a lot of output for a quickstart so I'm happy as long as it's demonstrated elsewhere in the docs.

Reply:
Let's stick with `kubectl get` for now - we can show `describe` in the more detailed guides.